Integrations/python/deephaven/transferrer/__init__.py

```diff
@@ -0,0 +1,81 @@
+# 
```

Comment:
I would probably move this to `learn/getter` or something like that.  I don't know that these are general-purpose methods, since they work on column sources and `learn.IndexSet`.  Also the name `transfer` is confusing when we use `getter` in the documentation.